make_kotlin.sh

```diff
@@ -0,0 +1,7 @@
+set -eu
+
+# Env
+UDL_NAME="iroh"
+
+# UniFfi bindgen
+cargo run --bin uniffi-bindgen generate "src/$UDL_NAME.udl" --language kotlin  --out-dir ./kotlin
```

Comment:
```
b5 at number-0 in ~/code/iroh-ffi (feat-kotlin)
$ ./make_kotlin.sh
   Compiling iroh-ffi v0.1.0 (/Users/b5/code/iroh-ffi)
    Finished dev [unoptimized + debuginfo] target(s) in 3.67s
     Running `target/debug/uniffi-bindgen generate src/iroh.udl --language kotlin --out-dir ./kotlin`
Warning: Unable to auto-format iroh.kt using ktlint: Os { code: 2, kind: NotFound, message: "No such file or directory" }
```

Reply:
running `$ curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.50.0/ktlint && chmod a+x ktlint && sudo mv ktlint /usr/local/bin/` from the [kotlin install instructions](https://pinterest.github.io/ktlint/1.0.0/install/cli/) fixed the warning